Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+String apkTag = null
+
+node {
+
+    apkTag = (env.BRANCH_NAME == "master")? "0.0.${env.BUILD_NUMBER}" : "0.0.${env.BUILD_NUMBER}-${env.BRANCH_NAME}"
+  /*
+    parameters
+     {
+    string(name:'STORE_PASSWORD', defaultValue:'dinocloud123$', description: 'STORE_PASSWORD is the password use when we built keystore.')
+    string(name:'PATH_TO_KEYSTORE',defaultValue:'/home/ubuntu/dinocloud.keystore', description:'PATH_TO_KEYSTORE is the path where we have the keystore key')
+     }
+    */
+
+    stage('Checkout')
+    {
+        checkout scm
+    }
+
+    stage('Build APK')
+    /*In this stage install the dependencies */
+    {
+      sh "npm install"  /* Really important (it installs all your project dependencies) */
+      sh "cordova platform rm android" /* Remove the old platform */
```

Comment:
Why are we using the "cordova" command and not the "ionic cordova" one? This apk is working ok? The ionic prefix will add all the ionic support. If not, it will create a cordova-native application and will not work.